src/clover/load/load.py

```diff
@@ -798,14 +809,35 @@ def process_device_hourly_usage(
 
         logger.info("Hourly usage profile for %s successfully calculated.", device.name)
 
+        #Retrieve device laod time
```

Comment:
# :memo: Longer comment?
As this is a fairly complicated bit of logic, maybe a longer comment explaining what's going on? E.G.,
```python
# Loop through the device utilisations and extend by the device usage time, in hours, to
# create a new profile.